app/models/user.rb

```diff
@@ -1,8 +1,10 @@
 # Huginn is designed to be a multi-User system.  Users have many Agents (and Events created by those Agents).
 class User < ActiveRecord::Base
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable, :lockable,
-         :omniauthable
+  DEVISE_MODULES = [:database_authenticatable, :registerable,
+                    :recoverable, :rememberable, :trackable,
+                    :validatable, :lockable, :omniauthable,
+                    (ENV['ALLOW_UNCONFIRMED_ACCESS'] == 'false' ? :confirmable : nil)].compact
```

Comment:
Existing users won't have this set at all until they update their `.env` files, right?  So they'll be nil and this will behave as it does now, I think.  But, it's a bit odd that false != nil here.  Maybe we should call it `REQUIRE_CONFIRMED_ACCESS` with a default of `false`, which matches the value everyone has now, which is `nil`? Does this make sense?